msu/vanilla_mod/load.nut

```diff
@@ -3,6 +3,6 @@ local function includeFile( _file )
 	::MSU.includeFile("msu/vanilla_mod/", _file + ".nut");
 }
 includeFile("setup_vanilla_mod");
```

Comment:
remove this includeFile if it's only being used once imo

Reply:
Eh, doesn't matter imo. I just copied the setup as in other load files and it's ok e.g. if we wanna extend this file with other files in the future.